homeassistant/components/intent/timers.py

```diff
@@ -711,6 +727,7 @@ class StartTimerIntentHandler(intent.IntentHandler):
     slot_schema = {
         vol.Required(vol.Any("hours", "minutes", "seconds")): cv.positive_int,
         vol.Optional("name"): cv.string,
+        vol.Optional("command"): cv.string,
```

Comment:
Should this feature be reflected in the description ?

Reply:
Added a description